waltz-ng/client/data-types/data-type-list.html

```diff
@@ -0,0 +1,84 @@
+<div>
```

Comment:
No, I'm working on the list page atm, it's nearly done, just refactoring
away the tree into a component

On Thu, Aug 11, 2016 at 5:28 PM, David Watkins <notifications@github.com>
wrote:

> In waltz-ng/client/data-types/data-type-list.html
> <https://github.com/khartec/waltz/pull/417#discussion_r74455832>:
>
> > @@ -0,0 +1,84 @@
> > +<div>
>
> Ignoring as it's a placeholder - let me know if it needs review....
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/khartec/waltz/pull/417/files/e4bc887bdadc42f03d5e44e0876776ce30ea1a5c#r74455832>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AC452zTu3AiyTW5Rqb-QvrCjT5vQUQ5Cks5qe02egaJpZM4JiBOI>
> .
>



-- 


*Kamran Saleem*
t  +44 (0) 7732 078654
e  kamran@thinkincode.co.uk
w thinkincode.co.uk